apps/signing/serializers.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from rest_framework import serializers
+
+import amo
+from amo.helpers import absolutify
+from files.models import FileUpload
+
+
+class FileUploadSerializer(serializers.ModelSerializer):
+    active = serializers.SerializerMethodField('get_active')
+    api_url = serializers.SerializerMethodField('get_api_url')
+    files = serializers.SerializerMethodField('get_files')
+    passed_review = serializers.SerializerMethodField('get_passed_review')
+    processed = serializers.CharField(source='processed')
+    reviewed = serializers.SerializerMethodField('get_reviewed')
+    validation_results = serializers.SerializerMethodField(
+        'get_validation_results')
+    validation_url = serializers.SerializerMethodField('get_validation_url')
+
+    class Meta:
+        model = FileUpload
+        fields = [
+            'active',
+            'api_url',
+            'files',
+            'passed_review',
+            'processed',
+            'reviewed',
+            'valid',
+            'validation_results',
+            'validation_url',
+            'version',
+        ]
+
+    def __init__(self, *args, **kwargs):
+        self.version = kwargs.pop('version', None)
+        super(FileUploadSerializer, self).__init__(*args, **kwargs)
+
+    def get_api_url(self, instance):
```

Comment:
I don't think the name should be specific to the model. If an API is truly RESTful then it should be browsable and I think there should be a common place to look for the URL.

If you look at GitHub's API [issues](https://api.github.com/repos/mozilla/olympia/issues) have a `url` (and some more `<prefix>_url` ones) as do [organizations](https://api.github.com/orgs/mozilla).

I think this should just be `url`.